.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "recommendations": [
+        "streetsidesoftware.code-spell-checker-spanish",
```

Comment:
Maybe the spanish spell checker is not needed.

BTW we would use i18n for switch languages? or will not handle multiple languages?

Reply:
I think is fine to add i18n support, not sure how it work with handlebars, tough